src/ZendCodingStandard/ruleset.xml

```diff
@@ -201,232 +70,823 @@
+         each subsequent line is indented at least once. When doing so, the
+         first condition MUST be on the next line. Boolean operators between
+         conditions MUST always be at the beginning or at the end of the line,
+         not a mix of both. -->
+    <!-- checked by PSR12.ControlStructures.ControlStructureSpacing -->
+    <!-- checked by PSR12.ControlStructures.BooleanOperatorPlacement -->
+    <!-- checked by Squiz.ControlStructures.ControlSignature -->
+
+    <!-- 5.4 for -->
+
+    <!-- Expressions in parentheses MAY be split across multiple lines, where
+         each subsequent line is indented at least once. When doing so, the
+         first expression MUST be on the next line. The closing parenthesis and
+         opening brace MUST be placed together on their own line with one space
+         between them. -->
+    <!-- checked by PSR12.ControlStructures.ControlStructureSpacing -->
+    <!-- checked by PSR12.ControlStructures.BooleanOperatorPlacement -->
+    <!-- checked by Squiz.ControlStructures.ControlSignature -->
+
+    <!-- TODO: allow multiline for structure -->
+
+    <!-- 5.5 foreach -->
+
+    <!-- exclude these messages as they are already checked by PSR2.ControlStructures.ControlStructureSpacing -->
+    <!-- checked by Squiz.ControlStructures.ForEachLoopDeclaration -->
+
+    <!-- 5.6 try, catch, finally -->
+
+    <!-- Additional Zend Framework rules -->
+
+    <!-- All catch blocks MUST be reachable. -->
+    <rule ref="SlevomatCodingStandard.Exceptions.DeadCatch"/>
+
+    <!-- Catch blocks SHOULD catch `Throwable` instead of `Exception`. -->
+    <rule ref="SlevomatCodingStandard.Exceptions.ReferenceThrowableOnly"/>
+
+    <!-- 6. Operators -->
+
+    <!-- When space is permitted around an operator, multiple spaces MAY be
+         used for readability purposes. -->
+    <!-- All operators not described here are left undefined. -->
+
+    <!-- Additional Zend Framework rules -->
+
+    <!-- There MUST be at least one space on either side of an equals sign used
+         to assign a value to a variable. In case of a block of related
+         assignments, more spaces MUST be inserted before the equal sign to
+         promote readability. -->
+    <rule ref="Generic.Formatting.MultipleStatementAlignment">
+        <properties>
+            <property name="error" value="true"/>
+            <property name="maxPadding" value="50"/>
+        </properties>
+    </rule>
+    <rule ref="Squiz.WhiteSpace.OperatorSpacing">
+        <properties>
+            <property name="ignoreNewlines" value="true"/>
+        </properties>
     </rule>
-    <!-- Force rules for function phpDoc -->
-    <rule ref="PEAR.Commenting.FunctionComment">
+    <!-- There MAY NOT be any white space around the object operator unless
+         multilines are used. -->
+    <rule ref="Squiz.WhiteSpace.LogicalOperatorSpacing"/>
+    <!-- There MAY NOT be spaces around the object operator. -->
```

Comment:
```suggestion
    <!-- There MUST NOT be spaces around the object operator. -->
```

Reply:
Merged this with the previous one as it's the same sniff.